src/main/java/org/apache/commons/pool2/PoolUtils.java

```diff
@@ -307,6 +307,8 @@ public static void checkRethrow(final Throwable t) {
      * @param pool
      *            the ObjectPool to be "wrapped" in a synchronized ObjectPool.
      * @param <T> the type of objects in the pool
+     * @throws IllegalArgumentException
+     *             when <code>pool</code> is <code>null</code>.
```

Comment:
Please use `{@code ...}` instead of HTML tags, it is more succinct and less error prone.

Reply:
I used html tags because I saw the other Javadoc in this class are using it.  Do you want me to change them as well?

Reply:
That would be great :-)

Reply:
I made the change in the new commit, please take a look